federation/backends/collections.md

```diff
@@ -31,6 +43,7 @@ We list here a set of common properties, that can be relevant for multiple colle
 
 - [sat:orbit_state](https://github.com/stac-extensions/sat#satorbit_state)
 - [sat:relative_orbit](https://github.com/stac-extensions/sat#satrelative_orbit)
+- [sar:instrument_mode](https://github.com/stac-extensions/sar#item-properties-or-asset-fields)
```

Comment:
SAR instrument mode doesn't make sense for Sentinel-2, which is not SAR...

Reply:
From the hierarch I understand `Common properties` to be on the same level as Sentinel-2 collection so that they can be used on all/various collections (wherever applicable).

Reply:
True, I didn't see that "Common properties" is level 2. On the other hand, there is a dedicated section for Optical, so a separate Section for SAR would be consistent.

Reply:
> there is a dedicated section for Optical, so a separate Section for SAR would be consistent.

Makes sense, moved to a new "SAR instruments" section.